### ISSUE TITLE
Google Version Handler settings have verbose logging on by default, proceed to spam the console horribly

### ISSUE THREAD

**Issue by edelventhal** [googlesamples/unity-jar-resolver#391]
### Please fill in the following fields:
Unity editor version: 2019.2.17f1 Personal
External Dependency Manager version: 1.2.153
Source you installed EDM (from .unitypackage or Unity Package Manager): Google Play .unitypackage
Features in External Dependency Manager in use (Android Resolver, iOS Resolver, VersionHandler, etc.): All of them seem turned on by default
Plugins SDK in use (Firebase, Admob, Facebook, etc.): Steamworks.NET
Platform you are using the Unity editor on (Mac, Windows, or Linux): Mac
Platform you are targeting (iOS, Android, and/or desktop): iOS, Android, Mac, PC, Linux
Scripting Runtime (Mono, and/or IL2CPP): IL2CPP

### Please describe the issue here:

<img width="296" alt="Screen Shot 2020-07-16 at 1 53 26 PM" src="https://user-images.githubusercontent.com/1554905/87721726-c2b08d80-c76b-11ea-971c-555b681a6fad.png">

By default, verbose logging is turned on here. Given that there are multiple settings panels and they are not in an obvious location, and given that the External Dependency Manager was installed as a byproduct of needing access to the Google Play APIs, there was no reason whatsoever for me to expect to find the verbose logging option here. Because of it, every single time I hit Play, made changes to code files, or made a build, my console was spammed with irrelevant information that drowned out actual messages I needed to see. It took me multiple days to find out how to turn it off.

<img width="558" alt="Screen Shot 2020-07-16 at 1 55 23 PM" src="https://user-images.githubusercontent.com/1554905/87721983-2f2b8c80-c76c-11ea-870e-017e62cfe2a5.png">

Verbose logging should be off by default, and you should also strongly consider whether you need to be running the dependency manager so frequently. It's a hidden waste of CPU cycles that is costing me and other developers precious time (see other issues reported here).

### Please answer the following, if applicable:
What's the issue repro rate? (eg 100%, 1/5 etc)

100%

**Comment by alexames** [comment]
Hi @edelventhal 

It looks like Verbose logging should be disabled by default (see [here](https://github.com/googlesamples/unity-jar-resolver/blob/master/source/VersionHandlerImpl/src/VersionHandlerImpl.cs#L2507)). It's possible that this flag was inadvertently turned on at some point in the past and stored in the project settings `GvhProjectSettings.xml`. Can you verify if the flag is set in that file? If there is a verbose logging entry in that file, that means it would had to have been explicitly turned on at some point.

Can you verify whether this issue still occurs when you create a clean project with a fresh install of EDM.

**Comment by edelventhal** [comment]
While it's _possible_ I toggled the verbose flag myself, I really don't think I did. I didn't even find those panels until later. However, it's good to know I was the exception.

For what it's worth, I installed the version handler via this: https://github.com/playgameservices/play-games-plugin-for-unity/blob/master/current-build/GooglePlayGamesPlugin-0.10.09.unitypackage

And, looking at my git history, the verbose setting was _not_ on immediately... it somehow turned on when I _deleted_ the Google Play and External Dependency Manager from my project (because I was getting dll conflicts from an old install). I did a manual deletion of the folders, and doing that somehow edited the `GvhProjectSettings.xml` file, changing the file in this way:

`-  <projectSetting name="Google.VersionHandler.VerboseLoggingEnabled" value="False" />
+  <projectSetting name="Google.VersionHandler.VerboseLoggingEnabled" value="True" />
+  <projectSetting name="Google.VersionHandler.VersionHandlingEnabled" value="True" />
+  <projectSetting name="GooglePlayServices.PromptBeforeAutoResolution" value="False" />`

Somewhere along the way of trying to fix the installation issues I also installed and deleted the External Dependency Manager from the Unity Package Manager, and what finally sorted me out was to use the uninstall menu items, which did a clean uninstall, then reinstalled it.

However, that verbose setting, which I hadn't touched, managed to remain. No idea if any of that means anything to you, but hopefully it helps.

Thanks, Eli
